contract_sale_generation/models/contract.py

```diff
@@ -32,8 +32,12 @@ def _prepare_sale(self, date_ref):
             sale.payment_term_id = self.payment_term_id.id
         if self.fiscal_position_id:
             sale.fiscal_position_id = self.fiscal_position_id.id
-        # Get other sale values from partner onchange
-        sale.onchange_partner_id()
+        # Get other sale values
+        sale._compute_partner_shipping_id()
```

Comment:
Should we really call the computes ? Wouldn't the values automatically computed ?